FiveCalls/FiveCalls/AboutSheet.swift

```diff
@@ -0,0 +1,140 @@
+                        EmailComposerView() { _ in }
+                    })
+                    .alert(isPresented: $showEmailComposerAlert) {
+                        Alert(title: Text(R.string.localizable.cantSendEmailTitle()),
+                              message: Text(R.string.localizable.cantSendEmailMessage()),
+                              dismissButton: .default(Text(R.string.localizable.dismissTitle())))
+                    }
+                    AboutListItem(title: R.string.localizable.aboutItemShowWelcome()) {
+                        showWelcome = true
+                    }
+                    .sheet(isPresented: $showWelcome, content: {
+                        WelcomeView()
+                    })
+                }
+                
+                Section(header: Text(R.string.localizable.aboutSectionHeaderSocial()),
+                        footer: Text(R.string.localizable.aboutSectionFooterSocial()))
+                {
+                    AboutListItem(title: R.string.localizable.aboutItemFollowTwitter()) {
+                        followOnTwitter()
+                    }
+                    if appUrl != nil {
+                        AboutListItem(title: R.string.localizable.aboutItemShare(), url: appUrl)
+                    }
+                    AboutListItem(title: R.string.localizable.aboutItemRate()) { requestReview() }
+                }
+                
+                if let versionString {
+                    Section(
+                        header: HStack {
+                            Spacer()
+                            Text(versionString)
+                                .font(.caption)
+                                .foregroundStyle(.gray)
+                            Spacer()
+                        },
+                        content: {})
+                }
+            }
+            .listStyle(.grouped)
+            .navigationTitle(R.string.localizable.aboutTitle())
+            .navigationBarTitleDisplayMode(.inline)
+            .navigationBarBackButtonHidden()
+            .toolbarBackground(.visible)
+            .toolbarBackground(Color.fivecallsDarkBlue)
+            .toolbarColorScheme(.dark, for: .navigationBar)
+            .toolbar {
+                ToolbarItem(placement: .navigationBarTrailing) {
+                    Button(action: {
+                        self.dismiss()
+                    }) {
+                        Text(R.string.localizable.doneButtonTitle())
+                            .bold()
+                    }
+                }
+            }
+        }
+        .accentColor(.white)
+    }
+    
+    func followOnTwitter() {
+        AnalyticsManager.shared.trackEventOld(withName: "Action: Follow On Twitter")
+        guard let url = URL(string: "https://twitter.com/make5calls") else { return }
+        UIApplication.shared.open(url)
+    }
+
+    
+    func requestReview() {
+        AnalyticsManager.shared.trackEventOld(withName: "Action: Rate the App")
```

Comment:
I didn't add these to the visually-new screens yet but it's good that you've added them here already!